entity-service-api/src/main/proto/org/hypertrace/entity/query/service/v1/entity_query_request.proto

```diff
@@ -25,6 +25,19 @@ message EntityUpdateRequest {
   repeated Expression selection = 4;
 }
 
+message BulkUpdateRequest {
+  string entityType = 1;
+  map<string, EntityUpdateInfo> bulkUpdateMap = 2;
+  message EntityUpdateInfo {
+    repeated UpdateInfo updateInfo = 1;
+  }
+}
+
+message UpdateInfo {
+  UpdateOperation updateOperation = 1;
+  repeated Expression selection = 2;
```

Comment:
`repeated Expression selection = 2;` shouldn't be part of `UpdateInfo`. It should be part of `BulkUpdateRequest`